bom/pom.xml

```diff
@@ -46,9 +46,9 @@
     <spring.version>4.3.2.RELEASE</spring.version>
     <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
     <truth.version>0.28</truth.version>
-    <vertx.version>3.3.3</vertx.version>
-    <vertx-proton.version>3.4.0.Beta1</vertx-proton.version>
-    <vertx-mqtt.version>3.4.0.Beta1</vertx-mqtt.version>
+    <vertx.version>3.4.0</vertx.version>
+    <vertx-proton.version>3.4.0</vertx-proton.version>
```

Comment:
We do not have a CQ for vertx-proton 3.4.0. However, we do have an approved CQ for 3.4.0.Beta1 so we should revert to that version for the time being. And make sure to create a CQ for 3.4.0 ... :-)